src/org/yccheok/jstock/gui/JStock.java

```diff
@@ -470,6 +470,11 @@ public void stateChanged(javax.swing.event.ChangeEvent evt) {
         this.jTable1.getTableHeader().addMouseListener(new TableColumnSelectionPopupListener(1));
         this.jTable1.addMouseListener(new TableRowPopupListener());
         this.jTable1.addKeyListener(new TableKeyEventListener());
+        jTable1.addMouseListener(new java.awt.event.MouseAdapter() {
+            public void mouseClicked(java.awt.event.MouseEvent evt) {
+                jTable1MouseClicked(evt);
+            }
+        });
```

Comment:
I had change it to handle in one place. Does the code change make sense to you?

https://github.com/yccheok/jstock/blob/master/src/org/yccheok/jstock/gui/JStock.java#L4405